ilastik/applets/dataSelection/datasetDetailedInfoTableModel.py

```diff
@@ -37,15 +38,12 @@ class DatasetColumn:
     NumColumns = 6
 
 
-def _resolution_to_display_string(resolution: List[int], axiskeys: str) -> str:
-    """
-    In Precomputed format, resolution is in xyz order, which we want to display in the same order as axiskeys.
-    To support formats other than Precomputed, the tableModel would need to be able to obtain
-    the order of the resolution axes (or generally, how to transform resolution to a display string)
-    from the datasetSlot.
-    """
-    assert len(resolution) == 3, "Expected resolution to be in xyz order, please report this on http://image.sc"
-    input_axes = dict(zip("xyz", resolution))
+def _scale_to_display_string(scale: Multiscale, axiskeys: str) -> str:
+    """Generate labels to put into the scale combobox.
+    scale.resolution must be in xyz and will be reordered to match axiskeys."""
+    if scale.resolution is None:
+        return scale.key
```

Comment:
It's leftover from an earlier state before I decided to pass the shape as resolution to the GUI in the case of OME-Zarr. We can remove the case and make `.resolution` mandatory. Maybe should rename this to `dimensions` since it's kind of just "Whatever info related to the dimensions at that scale level for the GUI".